srcpkgs/ccache/template

```diff
@@ -15,11 +13,31 @@ changelog="https://ccache.dev/releasenotes.html"
 distfiles="https://github.com/ccache/ccache/releases/download/v${version}/${pkgname}-${version}.tar.xz"
 checksum=ac1b82fe0a5e39905945c4d68fcb24bd0f32344869faf647a1b8d31e544dcb88
 
+if [ -z "$CHROOT_READY" ]; then
+	hostmakedepends="cmake-bootstrap"
+	# tests require ctest, not available in cmake-bootstrap
+	configure_args+=" -DENABLE_TESTING=OFF"
+else
+	hostmakedepends="asciidoc cmake perl"
```

Comment:
Shouldn't perl still be in `checkdepends`?

Reply:
no